crates/compiler/builtins/bitcode/build.zig

```diff
@@ -119,9 +119,9 @@ fn generateObjectFile(
 
     const suffix =
         if (target.result.os.tag == std.Target.Os.Tag.windows)
-        "obj"
-    else
-        "o";
+            "obj"
+        else
+            "o";
```

Comment:
This change is in the old compiler and probably wrong?

I would guess that zig 0.13.0 filters this differently than zig 0.14.0

Reply:
Yeah it was zig 0.14.0 that made this change. But old-workflows have ran this over and been happy so I left it.